packages/serverless-cdk-plugin/package.json

```diff
@@ -63,9 +61,15 @@
     "prettier": "^2.7.1",
     "serverless": "^3.19.0",
     "ts-node": "^10.8.1",
+    "ts-toolbelt": "^9.6.0",
     "ttypescript": "^1.5.13",
     "typescript": "^4.7.4"
   },
+  "exports": {
+    "./helpers": "./dist/cjs/helpers.js",
+    ".": "./dist/cjs/index.js"
+  },
+  "moduleResolution": "Node16",
```

Comment:
Are these necessary?

Reply:
We want our plugin to export two components: the serverless plugin as well as a helper function to facilitate autocomplete for construct attributes.

We were unable to export both the serverless plugin and a helper function in our `index.ts `:

This works:
```ts
import { ServerlessCdkPlugin } from './serverlessCdk';

module.exports = ServerlessCdkPlugin;
```

While this does not work:
```ts
import { ServerlessCdkPlugin } from './serverlessCdk';
import { getCdkProperty } from './helper';

module.exports = {ServerlessCdkPlugin, getCdkProperty};
```

We found that this exotic approach solves our issue.